### DISCUSSION TITLE
left_drawer overlay

### DISCUSSION THREAD

**Discussion by luke504** [zauberzeug/nicegui#2047] [Q&A]
### Question

brother you need to explain something to me: how is it possible that the left_drawer does not have the prop overlay?

**Comment by phoskee** [comment]
I'm not sure if I properly understand what you need, maybe `fixed=False` can help you?

**Reply by luke504** [reply]
my buddy doesn't work, who can I tell you?

**Comment by falkoschindler** [comment]
Yes, there is no initializer parameter called "overlay". This is because we try to provide the most important ones, but adding _all_ props would clutter the API without providing much value.

As with other elements, you can always add props like this:
```py
ui.left_drawer().props('overlay')
```

**Reply by luke504** [reply]
i clearly understood that it can't be done with your framework, i came to the point of making my sidebar in a custom way, using html and javascript. my main problem is that i couldn't render the sidebar in an overlapping way

**Reply by falkoschindler** [reply]
But it _can_ be done, as shown in my example code.

**Reply by phoskee** [reply]
Non facciamoci riconoscere.
If you can provide example code and what you expect, it's easier to find a solution togheter
